databricks/koalas/tests/test_dataframe.py

```diff
@@ -569,3 +569,45 @@ def test_sample(self):
             kdf.sample()
         with self.assertRaises(NotImplementedError):
             kdf.sample(n=1)
+
+    def test_pivot_table(self):
+
+        pdf = pd.DataFrame({'a': [4, 2, 3, 4, 8, 6],
+                            'b': [1, 2, 2, 4, 2, 4],
+                            'c': [1, 2, 9, 4, 7, 4]},
+                           index=[10, 20, 30, 40, 50, 60])
+
+        kdf = ks.from_pandas(pdf)
+
+        msg = "values should be string or list of columns."
+        with self.assertRaisesRegex(ValueError, msg):
+            kdf.pivot_table(index=['c'], columns="a", values=5)
+
+        msg = "index should be a None or a list of columns."
+        with self.assertRaisesRegex(ValueError, msg):
+            kdf.pivot_table(index="c", columns="a", values="b")
+
+        pdf = pd.DataFrame({'a': [4, 2, 3, 4, 8, 6],
```

Comment:
Could this DataFrame not simply be defined in line 575?

Reply:
I checked it and it works. Thanks!